extensions/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts

```diff
@@ -489,7 +489,7 @@ export class BdcRouterApi {
      * @param data Cluster configuration in JSON format
      * @param {*} [options] Override http request options.
      */
-    public createCluster (xRequestId: string, connection: string, data: string, options: any = {}) : Promise<{ response: http.ClientResponse; body: any;  }> {
+    public createCluster (xRequestId: string, connection: string, data: string, options: any = {}) : Promise<{ response: http.IncomingMessage; body: any;  }> {
```

Comment:
The problem with this is it becomes much harder to generate this API and just update automatically, rather than having to manually edit / maintain it all the time. Is there another way to fix this? E.g. could we alias this at the top somehow? Or start using a better code generator such that it uses the newer types here? I was using the online version but using a local one installed via NPM might solve?

Reply:
I can look into this. I don't really think it's that big of a deal though - you'll get compile errors when that happens and it's easy enough to do a replace all. I think the best option is to find a better tool (or better options for the existing tool) so I'll try to find some time to look into this.